lib/features/email/domain/usecases/parse_calendar_event_interactor.dart

```diff
@@ -22,26 +20,10 @@ class ParseCalendarEventInteractor {
     try {
       yield Right(ParseCalendarEventLoading());
 
-      final listResult = await Future.wait(
-        [
-          _calendarEventRepository.parse(accountId, blobIds),
-          _calendarEventRepository.getListEventAction(emailContents),
```

Comment:
IMO, also should delete `getListEventAction` function in repository and datasource when not used

Reply:
Done